contents/en/basics/import.md

```diff
@@ -99,30 +99,33 @@ For developers who are using TypeScript to develop ECharts, type interface is pr
 import * as echarts from 'echarts/core';
 import {
   BarChart,
-  // The series option types are defined with the SeriesOption suffix
-  BarSeriesOption,
   LineChart,
-  LineSeriesOption
 } from 'echarts/charts';
 import {
   TitleComponent,
-  // The component option types are defined with the ComponentOption suffix
-  TitleComponentOption,
   TooltipComponent,
-  TooltipComponentOption,
   GridComponent,
-  GridComponentOption,
   // Dataset
   DatasetComponent,
-  DatasetComponentOption,
   // Built-in transform (filter, sort)
   TransformComponent
 } from 'echarts/components';
 import { LabelLayout, UniversalTransition } from 'echarts/features';
 import { CanvasRenderer } from 'echarts/renderers';
+import type { 
+  ComposeOption, 
+  // The series option types are defined with the SeriesOption suffix
+  BarSeriesOption, 
+  LineSeriesOption,
+  // The component option types are defined with the ComponentOption suffix
+  TitleComponentOption, 
+  TooltipComponentOption,
+  GridComponentOption,
+  DatasetComponentOption
+} from 'echarts';
```

Comment:
I'm concerned about the import source. While it looks fine to import all types from `'echarts'`, will it violate the meaning of the on-demand import? Should we import them from different entries? Just like importing the component and series. I'm not sure.

```ts
import type {
  BarSeriesOption,
  LineSeriesOption
} from 'echarts/charts';

import type {
  TitleComponentOption,
  TooltipComponentOption,
  GridComponentOption,
  DatasetComponentOption
} from 'echarts/components';

import type { 
  ComposeOption
} from 'echarts/core';
```

If so, I think it's more complex than before, even though it looks more clear.

Reply:
Import by type should not affect on-demand import, but import from a different entry does make it clearer, I'll update.